profiles/common/modules/features/cce_basic_config/cce_basic_config.module

```diff
@@ -358,3 +358,191 @@ function cce_basic_config_preprocess_container(&$variables) {
     $variables['theme_hook_suggestions'][] = $prefix . $suffix;
   }
 }
+
+/**
+ * Implements hook_themes_enabled().
+ */
+function cce_basic_config_themes_enabled($theme_list) {
```

Comment:
After the discussion with the team, we decided not to use hook_themes_enabled. Instead, we will call the configuration methods on a dedicated drush method.